src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.vh.nursery;
+
+
+
+public class DefaultCountingOutRhymer {
+
+
+    private static final int MAX_CAP = 12;
+    private static final int EMPTY_STACK = -1;
+    private static final int LAST_ELEM = 11;
```

Comment:
This constant is related to capacity of numbers array. Changing only one leads to an error (inconsistency).